contracts/interfaces/IUSDC.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.2;
+
+interface IUSDC {
+    function deposit() external payable;
+
+    function withdraw(uint256) external;
```

Comment:
USDC contract doesn't have `deposit` and `withdraw` functions

Reply:
Missing out the `mint` function in here for minting in tests

Reply:
instead of mint I just transfer usdc from the largest holder (couldn't find minter for usdc)